src/Lusid.Sdk.Examples/tutorials/MarketData/CorporateActions.cs

```diff
@@ -423,8 +423,8 @@ public void Process_Name_Change_By_Transitions()
 
             try
             {
-                _apiFactory.Api<IInstrumentsApi>().DeleteInstrument("Figi", originalInstrument.Figi);
-                _apiFactory.Api<IInstrumentsApi>().DeleteInstrument("Figi", newInstrument.Figi);
+                _apiFactory.Api<IInstrumentsApi>().DeleteInstrument("Figi", originalInstrument.ClientInternal);
+                _apiFactory.Api<IInstrumentsApi>().DeleteInstrument("Figi", newInstrument.ClientInternal);
```

Comment:
Should this be `ClientInternal` and not `Figi`

Reply:
good spot